someSong/src/pages/home/home.ts

```diff
@@ -34,6 +36,9 @@ export class HomePage {
       this._user.getUserQuestions(this.user.userID).on('child_added', userQuestion => {
         this._question.getQuestionDetails(userQuestion.key).subscribe((questionDetail) => {
           this.userQuestions = DictionaryHelpFunctions.addToDictionary(this.userQuestions, userQuestion.key, questionDetail);
+
+          this.questionLoading = false;
```

Comment:
ok.. thats right, i didnt check it..
but then how can I know if the client finnished loading from firebase?